cudamapper/src/overlapper_triggered.cu

```diff
@@ -166,6 +166,6 @@ namespace claragenomics {
         }
 
         //Return fused overlaps
-        return fuse_overlaps(overlaps);
+        fuse_overlaps(fused_overlaps, overlaps);
```

Comment:
If this function is now `void` then this comment is a tiny bit confusing, it is writing to `&fused_overlaps` really?